docs/table-of-contents.json

```diff
@@ -46,7 +46,7 @@
         },
         {
           "name": "Using with other Frameworks",
-          "content": "get-started/using-with-other-frameworks"
```

Comment:
this content vs entry is the main reason why the TOC would not render well in gatsby

Reply:
Unless we can make ocular generate warnings (might be hard for us to detect since the error presumably occurs during the gatsby graphql "magic" processing phase), maybe we need a page with "website debugging tips" to our ocular-gatsby docs.

E.g. if pages are missing, check for this...

Reply:
Right. This is a problem for the ocular package though. Ocular does emits warnings when it cannot find a corresponding node for a TOC entry but not the other way round.